templates/taxi/driver_list.html

```diff
@@ -20,6 +21,10 @@ <h1>Driver List
         <td>{{ driver.first_name }}</td>
         <td>{{ driver.last_name }}</td>
         <td>{{ driver.license_number }}</td>
+        <td>
+          <a style="color: red"
+             href="{% url 'taxi:driver-delete' pk=driver.id %}">Delete</a>
```

Comment:
The 'Delete' link has inline styling with `color: red`. Consider using a CSS class for styling instead of inline styles to maintain consistency and separation of concerns.